src/main/java/org/breedinginsight/brapi/v2/services/BrAPIGermplasmService.java

```diff
@@ -50,14 +50,23 @@ public BrAPIGermplasmService(BrAPIListDAO brAPIListDAO, ProgramService programSe
     }
 
     public List<BrAPIGermplasm> getGermplasm(UUID programId) throws ApiException {
-        List<BrAPIGermplasm> germplasmList;
         try {
             return germplasmDAO.getGermplasm(programId);
         } catch (ApiException e) {
             throw new InternalServerException(e.getMessage(), e);
         }
     }
 
+    public BrAPIGermplasm getGermplasmByUUID(UUID programId, String germplasmId) throws ApiException, DoesNotExistException {
```

Comment:
Given that you're catching an `ApiException`, you don't need to specify that an `ApiException` will be thrown.